gandalf.js

```diff
@@ -1,33 +1,32 @@
-/**
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.